Source/WebCore/history/HistoryItem.h

```diff
@@ -57,30 +57,22 @@ class Image;
 class ResourceRequest;
 enum class PruningReason;
 
-WEBCORE_EXPORT extern void (*notifyHistoryItemChanged)(HistoryItem&);
+class HistoryItemClient : public RefCounted<HistoryItemClient> {
```

Comment:
Seems we could move this inside the `HistoryItem` class and rename to `Client`? At the very least, we could add a `using Client = HistoryItemClient;` inside `HistoryItem`.